django/conf/locale/tr/formats.py

```diff
@@ -14,7 +14,8 @@
 # The *_INPUT_FORMATS strings use the Python strftime format syntax,
 # see https://docs.python.org/library/datetime.html#strftime-strptime-behavior
 DATE_INPUT_FORMATS = [
-    '%d/%m/%Y', '%d/%m/%y',     # '25/10/2006', '25/10/06'
+    '%d/%m/%Y',  # '25/10/2006'
+    '%d/%m/%y',  # '25/10/06'
     '%y-%m-%d',                 # '06-10-25'
     # '%d %B %Y', '%d %b. %Y',  # '25 Ekim 2006', '25 Eki. 2006'
```

Comment:
Conscious decision to leave these as they are? I was wondering if they could be removed?

Reply:
> Conscious decision to leave these as they are?

Yes, it's only a comment not affected by `black`.

> I was wondering if they could be removed?

They can be helpful for folks that need additional formats. I'd leave them.